letter-sender/src/main/java/com/christmas/letter/sender/config/AwsConfig.java

```diff
@@ -0,0 +1,20 @@
+package com.christmas.letter.sender.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.sns.SnsClient;
+
+import java.net.URI;
+
+@Configuration
+public class AwsConfig {
+
+    @Bean
+    public SnsClient snsClient(){
+        return SnsClient.builder()
+                .region(Region.US_EAST_1)
+                .endpointOverride(URI.create("http://localhost:4566"))
+                .build();
```

Comment:
all the properties for the external dependent resources is recommended to be loaded from properties files. I would leverage application.properties for this

Reply:
you're right, fixed it